src/PgAsync/Client.php

```diff
@@ -50,12 +65,44 @@ public function query($s)
     public function executeStatement(string $queryString, array $parameters = [])
     {
         return Observable::defer(function () use ($queryString, $parameters) {
-            $conn = $this->getIdleConnection();
```

Comment:
Can we get rid of getIdleConnection?

Reply:
It is used in tests, but they could use reflection or something. Would this be a BC break? Maybe mark `@deprecated`?